io.github.Rubberduckycooly.RSDKv5.yaml

```diff
@@ -0,0 +1,57 @@
+app-id: io.github.Rubberduckycooly.RSDKv5
```

Comment:
the url `https://github.com/Rubberduckycooly/RSDKv5` is 404. Because that's not the name of the repository, therefore doesn't meet the requirements.

Reply:
the actual url includes hyphens, so it would've to be io.github.Rubberduckycooly.RSDKv5Decompilation or io.github.Rubberduckycooly.RSDKv5_Decompilation, which isn't a valid url either

Reply:
Re-read the documentation. The last component is allowed to have dashes. Also replacing `-` with `_` where `-` isn't allow is the common practice.

Reply:
Ok, changed to io.github.Rubberduckycooly.RSDKv5-Decompilation